conf/application.conf

```diff
@@ -193,6 +193,18 @@ scheduling {
       initialDelaySecs = 30
       intervalSecs = 30
     }
+    first-phase2-reminder-expiring-test-job {
+      enabled = false
+      lockId = "first-phase2-reminder-lock-coordinator"
+      initialDelaySecs = 31
```

Comment:
No... just thought not to have all of them starting at the same time. Shouldn't be a problem as long as the interval is the same.